app/services/user-settings.js

```diff
@@ -0,0 +1,67 @@
+import Object from '@ember/object';
+import Service from '@ember/service';
+import {getObject, setObject} from '../utils/storage';
+
+const SETTINGS_KEY = 'user_settings';
+const TOKENS_KEY = 'user_tokens';
+const PROPERTIES = ['githubTokenModalSeen'];
+
+// Use github_com instead of github.com for localstorage because of ember `set`
+// and `get` internals
+const userSettings = Service.extend({
+  init() {
+    this._super(...arguments);
+    this.tokens = Object.create({
+      'github_com': null,
```

Comment:
This isn't unreasonably confusing as-is, but why not just call it `github` instead of `github_com`? That would make it so we don't need the comment either, which just distracts from the code.

Reply:
We will have gitlab ce support. Which will fit the naming. e.g `gitlab_cern_ch`

Reply:
It lives in ember service, the property is exposed. Ember object has addtional `get` and `set` method. Doing `userSettings.get('tokens').set('gitlab_cern_ch')` is common in ember.

Reply:
I though github have similiar stuff, called github enterprise (Never saw one, but it one of its products).

Reply:
Okay, that works then